### PR TITLE
827156 UG documentation for the candle chart in .NET MAUI

### PR DESCRIPTION
**Pull request by NitheeshkumarThangaraj**
### Feature Description ###
Added the user guide document for candle series.

### Output ###
CandleBasicRendering
CandleFillColor
SolidCandles

### CODE REVIEW

File: MAUI/Cartesian-Charts/Candle.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Candle chart type in MAUI Chart](Chart-types_images/CandleBasicRendering.png)
+
+## Bull and Bear Color
+
+In the Candle chart, the [BullFillColor]() property is used to specify a fill color for the segments that indicates an increase in the stock price in the measured time interval, and the [BearFillColor]() property is used to specify a fill color for the segments that indicates a decrease in the stock price in the measured time interval.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Candle.md:89]: its BullishFill, BearishFill, change apis and code snippet too.

File: MAUI/Cartesian-Charts/Candle.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+title: Candle Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here all about candle chart support in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Candle Chart in .NET MAUI Chart
+
+Candlestick charts are a type of financial chart used to represent the price movement of an asset over time. The chart is made up of a series of rectangular bars, called candlesticks, that represent a specific time, typically one day. To render a candle chart, create an instance of [CandleSeries](), and add it to the [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) collection property of [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Candle.md:12]: Candle charts are a type of financial chart used to represent the price movement of an asset over time. The chart is made up of a series of rectangular bars, called candlesticks, that represent a specific time, typically one day.

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Candle.md (hunk 1 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Candle.md]: Changed

File: MAUI/Cartesian-Charts/Candle.md (hunk 2 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Cartesian-Charts/Candle.md]: Updated

**Review by Saravanan-Madhesh — APPROVED** [review]
The changes are fine.

**Pull request merged**